Sources/OpenTelemetrySdk/Metrics/Stable/LongCounterMeterBuilderSdk.swift

```diff
@@ -41,4 +41,14 @@ public class LongCounterMeterBuilderSdk: LongCounterBuilder, InstrumentBuilder {
         -> OpenTelemetryApi.ObservableLongCounter {
         registerLongAsynchronousInstrument(type: .observableCounter, updater: callback)
     }
+    
+    public func setDescription(_ description: String) -> LongCounterBuilder {
```

Comment:
setDescription & setUnit are accessible via the InstrumentBuilder protocol, remove these.